images/macos/toolsets/toolset-15.json

```diff
@@ -61,10 +61,10 @@
                     "version": "16.4.0+16F6",
                     "sha256": "2dbf65ba28fb85b34e72c14c529a42d5c3189ab0f11fb29fdebd5f4ee6c87900",
                     "install_runtimes": [
-                        { "iOS": ["18.0", "18.1", "18.2", "18.3.1", "18.4", "18.5"] },
-                        { "watchOS": ["11.0", "11.1", "11.2", "11.4", "11.5"] },
-                        { "tvOS": ["18.0", "18.1", "18.2", "18.4", "18.5"] },
-                        { "visionOS": ["2.0", "2.1", "2.2", "2.3", "2.4", "2.5"] }
+                        { "iOS": ["18.2", "18.4", "18.5"] },
+                        { "watchOS": ["11.2", "11.4", "11.5"] },
+                        { "tvOS": ["18.2", "18.4", "18.5"] },
+                        { "visionOS": ["2.2", "2.4", "2.5"] }
```

Comment:
This is definitely a good call. Thanks for keeping the "meaningful" versions instead of all the others that are effectively the same.

Great job adding the space for the newest betas, appreciated @erik-bershel 🚀 !